docs/guides/postinstall.md

```diff
@@ -100,6 +100,11 @@ The steps to be followed vary depending upon the initramfs module loading mechan
 
 ## Setting up the Touch Bar
 
+There's new touch bar software called tiny-dfr.  Below are instructions on how to install it on various distros.
+
+- Note: There is a bug causing tiny-dfr not to load in most distros if the file `/etc/tiny-dfr/config.toml` is blank.
```

Comment:
Isn't this bug fixed? Why adding then?

Reply:
It hasn't been fixed yet.  Or if it has, the fix isn't merged yet.  I'm happy to remove these lines after the fix is merged.